### ISSUE TITLE
Find another solution for ::-webkit-input-placeholder?

### ISSUE THREAD

**Issue by RhodesPeter** [FAC10/week3-PAMF#73]
When I looked into ::-webkit-input-placeholder to see what it does, I found this: 

![screen shot 2017-03-10 at 10 29 51](https://cloud.githubusercontent.com/assets/18164707/23791882/ae417850-057c-11e7-876a-aab0cda85b42.png)

Is there another way that you can do this which is more standard across browsers?

**Comment by oliverjam** [comment]
It looks like the browser support for styling placeholders in general isn't too bad (http://caniuse.com/#search=%3A%3Aplaceholder), but you need to include more prefixes than just the webkit one.

If you add `:-ms-input-placeholder` and `::-ms-input-placeholder` you'll also support IE and Edge.

**Comment by pbywater** [comment]
Thanks guys - I'd added the others but they seem not to work when you reference them but separating them with commas so I'd removed and forgotten to add again. Will do now!

**Comment by oliverjam** [comment]
Yeah you can't comma separate browser prefixed properties because of the way CSS is parsed — if a browser hits a declaration it doesn't understand (e.g. Chrome trying to read `-ms-`) it will just stop reading that block of code entirely
